docs/using_deis/using-buildpacks.rst

```diff
@@ -135,6 +135,14 @@ To use a custom buildpack, set the ``BUILDPACK_URL`` environment variable.
     === humble-autoharp
     BUILDPACK_URL: https://github.com/dpiddy/heroku-buildpack-ruby-minimal
 
+.. note::
+
+    If, however, you're unable to deploy using the latest version of the buildpack,
+    You can set
+    an exact version of a buildpack by using a git revision in your
+    BUILDPACK_URL.
+    git://repo.git#master git://repo.git#v1.2.0
```

Comment:
Could we change this to something like "For example: " and then render both BUILDPACK_URL and the URL itself as code blocks by using two backticks?